plugin/trino-iceberg/pom.xml

```diff
@@ -341,6 +341,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
This shouldn't be needed, as it's already a direct dependency, but I don't see anything in the code that would make it required at runtime.